src/main/keep-container/keep-content/keep-add-notes/keep-add-notes.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, ElementRef, HostListener, ViewChild } from '@angular/core';
+import { NoteService } from "../../../Service/note.service";
+
+@Component({
+  selector: 'app-keep-add-notes',
+  templateUrl: './keep-add-notes.component.html',
+  styleUrls: ['./keep-add-notes.component.scss'],
+
+})
+export class KeepAddNotesComponent {
+  showFirstForm: boolean = true;
+  showSecondForm: boolean = false;
+  showDropdownMenu: boolean = false;
+  showLabelDropdown: boolean = false;
+  title: string = '';
+  noteMessage: string = '';
+  @ViewChild('titleInput', {static: false}) titleInput!: ElementRef;
+  @ViewChild('secondForm') secondFormElement!: ElementRef;
+  @ViewChild('firstForm') firstFormElement!: ElementRef;
+  @ViewChild('form') FormElement!: ElementRef;
+  @ViewChild('textarea') Text!: ElementRef;
```

Comment:
we can following a template like the following:
first we should declare @ViewChild/@Input/@Output 
then we can declare our component level local variables.

Reply:
updated